Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:3.3.3-slim
+FROM ruby:3.3.5-slim
```

Comment:
I didn't see this in the list of [supported images ](https://engportal.ezcater.net/docs/default/component/container-base-images/base-images/ruby/) - does `3.3.5-alpine` give us everything that we need? IIRC there might be additional overhead because you might need to "manually" install some packages/libraries that we currently get for free with the `slim` image

Reply:
This is a good call-out.

Since this repo is forked from a public repo I don't believe we have the same docker base images available to us. Instead what I've done (and what has been done historically in this repo) is to use a base image from [this list of images](https://hub.docker.com/_/ruby)

Reply:
Does this mean the security dashboard/system consistently dings us for using a non-compliant image when theres nothing we can do about it? 🤔

Reply:
We technically can do something about it by updating the base image that is being used. Now, if one of the gems in the Gemfile was getting us dinged, we potentially have a bigger issue if it breaks the way slate works (i.e. upgrading the Haml gem).